permission-portal-frontend/client/src/store/index.js

```diff
@@ -1,15 +1,18 @@
-import { types, flow } from 'mobx-state-tree'
+import { types, flow , onSnapshot, getRoot, getSnapshot} from 'mobx-state-tree'
 import { firebase } from '../components/Firebase'
 
+
 const User = types
   .model({
-    uuid: types.string,
+    uuid: types.string, 
```

Comment:
Why do we need both `uuid` and `uid`?

Reply:
the hydration of the model was choking when we don't declare the database schema. I am will test this with removing uid or uuid. Do you have a preference on which one to keep?

Reply:
na same meaning, if this makes it easier to pass other objects into the store then its fine to have the redundancy, just make some note of why its there

Reply:
done. kept uuid because that is guaranteed and Native to firebase. the uid is application specific attribute and at times it is not populated. removing this in the latest commit